packaging/hudi-flink-bundle/pom.xml

```diff
@@ -107,6 +107,7 @@
                   <include>com.fasterxml.jackson.core:jackson-databind</include>
                   <include>com.fasterxml.jackson.core:jackson-core</include>
 
+                  <include>com.lmax:disruptor</include>
```

Comment:
I don't think we need to do the relocation for it, since the collision on it is unlikely. Relocation is very specific instrument which has its own strings attached to it and we should limit its usage only to cases when collisions are likely